### ISSUE TITLE
Browser Offer triggered by Incoming URL

### ISSUE THREAD

**Issue by danielc1234** [LiveHelperChat/livehelperchat#665]
So it would be nice to have a variable set that if we wanted the Browser Offer to fire off, the you could set a URL parameter of where the person came from.

So like it they came from an Adwords campaign, a certain browser offer would show.

Can this be accomplished?

**Comment by remdex** [comment]
Yes but will have to wait. As i'm just overloaded past few weeks with commercial stuff...

**Comment by PeopleInside** [comment]
remdex sorry if i write here but I'M happy to hear you have work as commercial :) is a very good thing and I'm happy to hear that. Your chat software are awesome and ... it's more beautiful of all software because is open source so congratulations to all people who decide to use commercial support and use this chat for business activities.
